app/prometheus/readme.md

```diff
@@ -0,0 +1,3 @@
+# What is prometheus
+
+Prometheus is a monitoring tool that we use to get data into our dashboards and alerts on [https://couchers.grafana.net/]. Prometheus works by scraping metrics from its data sources. In our case, the app respondes to GET requests on port 8000 and 8001 with a list of numbers and tags. Prometheus regularly calls these endpoints and then forwards the data to a centralized database we use (for FREE) at Grafana Cloud. From there we can use the data for monitoring and alerting.
```

Comment:
responds*

Not sure if it's necessary to tout the freeness... It won't be free for very long, it's not like we get some special treatment here?

Reply:
Will fix the typo and remove the freeness thing. Not relevant to the readme. 

The setup is claimed to be 'free forever' within (reasonably generous) usage limits.